sdk/containerregistry/Microsoft.Azure.Management.ContainerRegistry/src/Microsoft.Azure.Management.ContainerRegistry.csproj

```diff
@@ -6,12 +6,11 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.ContainerRegistry</PackageId>
 		<Description>Provides management capabilities for Azure Container Registry.</Description>
-		<Version>5.0.0-preview.5</Version>
+		<Version>6.0.0</Version>
 		<AssemblyName>Microsoft.Azure.Management.ContainerRegistry</AssemblyName>
 		<PackageTags>Microsoft Azure Container Registry;Container Registry;</PackageTags>
 		<PackageReleaseNotes>
 			<![CDATA[
-        1) Add Location property to ExportPipeline and ImportPipeline
```

Comment:
No release note for this version?

Reply:
Updated PR with release notes, as well as API tags. Thanks for the feedback.